tensorboard/webapp/runs/views/runs_table/types.ts

```diff
@@ -41,6 +41,7 @@ export interface RunTableItem {
   experimentAlias: ExperimentAlias;
   experimentName: string;
   selected: boolean;
+  runColor: string;
```

Comment:
Apparently this has just been wrong for years.

Reply:
I honestly do not understand how typescript did not catch this.